Diffing_Engine/Compute/Diffing.cs

```diff
@@ -0,0 +1,168 @@
+﻿using BH.oM.Base;
```

Comment:
One last thing is very minor. I think `Compute.Diff` rather than `Compute.Diffing` would be more appropriate? (Using the verb rather than the noun).

With diffing it comes easier for me `Create.Diffing`, but I think you're right, that's a `Compute`

Reply:
> One last thing, very minor. I think `Compute.Diff` rather than `Compute.Diffing` would be more appropriate? (Using the verb rather than the noun).

That's curious, to me [Diff is the noun](https://en.wiktionary.org/wiki/diff#English) and [Diffing is the verb](https://en.wiktionary.org/wiki/diffing) 😄 
At least that's how we've been using it during our conversations, I believe. I'd still keep it like that

Reply:
I think diffing is the gerund/infinite?

What I meant is that "I diff A and B to achieve  C` .

I am really not the best one to talk about this anyway, I am not that good in English.

Reply:
Me neither, we definitely need a native here 😄